### ISSUE TITLE
Unable to create project. All fields disappear

### ISSUE THREAD

**Issue by mayersj1** [marcusolsson/obsidian-projects#836]
### What happened?

Installed the Projects plugin.  Clicked to create new project.  After adding fields in, one of two things occur:
1. If I switch away from the project, when I come back to it, all the fields are gone
2. I also turned on 'Enable Rich Text Formatting' while creating a field and that caused all the fields to disappear
3. If I have created the fields and attempt to create a note, after prompting for the note name there is a new row that appears in the table view, but it is empty. it doesn't contain the note name.

The only thing that is saved is the project name and the views that I have selected.  Everything else is gone.

### What did you expect to happen?

I would expect this to work. if I use the demo project, I'm able to add notes to that successfully. I cannot edit the demo itself as going into any of the field configuration says that changing the field is not yet possible

### How can we reproduce it (as minimally and precisely as possible)?

I have no clue as I'm sure this isn't expected behavior.  What I haven't done is disable all other plugins that I have installed to see if there's a conflict there.

### Plugin version

1.17.2

### Obsidian version

1.5.12

### OS

macOS

**Comment by mayersj1** [comment]
I've turned off all community plugins except for the following:

Projects
Style Settings
Templater

Restricted mode is OFF

Core plugins disabled are:

Daily Notes
Outline
Publish
Random Note
Slides
Sync
Templates
Unique Note Creator
Word Count

**Comment by mayersj1** [comment]
Attempted to switch back to the Default theme and disable ALL community plugins except for Projects. Still seeing the same behavior

**Comment by mayersj1** [comment]
Just tried creating a new vault with all defaults and just added the Projects plugin. Still the same behavior.  Something isn't right here.

**Comment by Acylation** [comment]
Oh that should be an edge case. Without existing note the added field won’t persist. Could you please add a note first (via Projects table row or the Obsidian / file system, both are expected to work), then adding fields?

**Comment by mayersj1** [comment]
I've tried adding a note.  If you use the demo project, I can click the 'Add Note' button and the box asking for a note name comes up and once I enter a note name and click ok, the note name appears in the table.

When I'm trying to create a note in the project I'm creating a new blank row is created but it doesn't contain the name of the note. There is nothing there.

**Comment by mayersj1** [comment]
Seems like a pretty likely edge case as there is nothing in the documentation noting that you cannot create the project structure without adding a note.

**Comment by mayersj1** [comment]
Finally was able to get this working. The key is to create the folder location first.  My thought was that setting the project folder in the project creation dialog would create it. It does not.  I had to create the folder hierarchy first, then SELECT it from the list of available directories.

**Comment by Acylation** [comment]
Yeah there's a need to enhance the onboarding / guiding docs. As for the request on creating new folder if it is not yet created upon building a new project, this is logged in #663.

**Comment by alothings** [comment]
I have a very similar problem. 

- I created a new folder, I created a new project.
- In the first table I started adding fields.
- When I create a new field called stage
  - type text, default value "backlog"
  - Options: concept, studying, reviewing, done
  - click Create field
Then all other fields disappear. Why is the projects plugging so finicky? A previous time, an entire project disappeared, I was able to recover it after hours but now I am terrified of using it for something serious.
